lib/features/family/features/reflect/presentation/widgets/secret_word_input.dart

```diff
@@ -35,9 +37,11 @@ class _SecretWordInputState extends State<SecretWordInput> {
       children: [
         for (var i = 0; i < widget.amountOfLetters; i++) ...[
           Padding(
-            padding: const EdgeInsets.fromLTRB(2, 0, 2, 0),
+            padding: isLongWord
+                ? const EdgeInsets.fromLTRB(0, 0, 1, 0)
+                : const EdgeInsets.fromLTRB(2, 0, 2, 0),
             child: SizedBox(
-              width: 24,
+              width: isLongWord ? 20 : 24,
```

Comment:
I wouldnt change this based on the length of the words. Letter M (and maybe some others like w?) doesn't fit with 20. 

Was talking on Friday a bit with Andy about this and I think we just need to remove the 2 longest words and keep a maximum length for now

Reply:
The long words 'friendship' and 'adventure' do not contain the letter m. 
moon, smile, home and jump have it in the list, and they are all short so this works.